socket.template.ini

```diff
@@ -0,0 +1,165 @@
+name = "PlayerchainDemo"
+output = "build"
+copy = "dist"
+; script = "node ...."
+
+
+[build.script]
+forward_arguments = false
+
+
+[build.watch]
+; sources[] = "src"
+
+
+[webview]
+root = "/"
+watch = false
+; headers[] = "Cross-Origin-Opener-Policy: same-origin"
+; headers[] = "Cross-Origin-Embedder-Policy: require-corp"
+
+
+
+[webview.watch]
+reload = false
+; service_worker_reload_timeout = 500
+
+
+[webview.navigator.mounts]
+; $HOST_HOME/directory-in-home-folder/ = /mount/path/in/navigator
+; $HOST_CONTAINER/directory-app-container/ = /mount/path/in/navigator
+; $HOST_PROCESS_WORKING_DIRECTORY/directory-in-app-process-working-directory/ = /mount/path/in/navigator
+
+
+[webview.navigator.policies]
+; allowed[] = "https://*.example.com/*"
+
+
+[permissions]
+allow_fullscreen = true
+; allow_microphone = true
+; allow_camera = true
+; allow_user_media = true
+; allow_geolocation = true
+; allow_notifications = true
+; allow_sensors = true
+allow_clipboard = true
+; allow_bluetooth = true
+; allow_data_access = true
+; allow_airplay = true
+; allow_hotkeys = true
+
+
+[debug]
+flags = "-g"
+
+
+[meta]
+bundle_identifier = "com.playmint.playerchaindemo"
+application_protocol = "playerchaindemo"
+file_limit = 1024
+lang = "en-us"
+maintainer = "Playmint"
+title = "PlayerchainDemo"
+version = 1.0.0
+
+
+[android]
+aapt_no_compress = ""
+enable_standard_ndk_build = false
+main_activity = ""
+manifest_permissions = ""
+native_abis = ""
+native_cflags = ""
+native_sources = ""
+native_makefile = ""
+sources = ""
+icon = "src/icon.png"
+icon_sizes = "512@1x"
+
+
+[ios]
+codesign_identity = ""
+distribution_method = "release-testing"
+provisioning_profile = ""
+simulator_device = "iPhone 14"
+; nonexempt_encryption = false
+icon = "src/icon.png"
+icon_sizes = "29@1x 29@2x 29@3x 40@2x 40@3x 57@1x 57@2x 60@2x 60@3x"
+
+
+[linux]
+categories = "Developer Tools"
+icon = "src/icon.png"
+icon_sizes = "512@1x"
+
+
+[mac]
+category = ""
+codesign_identity = "46FB07C72E1D9065E6D3F7B8AAE8CA488C318DC1"
+codesign_paths = ""
+minimum_supported_version = "14.0.0"
+; window_control_offsets = "10x24"
+icon = "src/assets/icons/app-macos-512x512-2x.png"
+icon_sizes = "16@1x 32@1x 128@1x"
+
+[apple]
+team_identifier = "V893S45NX5"
+
+[native]
+; files = native-module1.cc native-module2.cc
+; headers = native-module1.hh
+
+
+[win]
+logo = "app-win-512x512-2x.png"
+icon = "src/assets/icons/app-win-512x512-2x.png"
+icon_sizes = "512@1x"
+; !! when the real certificate is added, the publisher value     !!
+; !! must be updated to match the certificate publisher value    !!
+publisher = "CN=PLAYMINT LIMITED, O=PLAYMINT LIMITED, L=Brighton, S=Brighton and Hove, C=GB"
+; pfx = "certs/cert.pfx"
+
+
+[window]
+width = 1152
+height = 768
+min_width = 640
+min_height = 480
+resizable = true
+; backgroundColorDark = "rgba(0, 0, 0, 1)"
+; backgroundColorLight = "rgba(255, 255, 255, 1)"
+<% if (platform === 'macos' || platform === 'linux') { %>titlebar_style = "hidden"<% } %>
```

Comment:
since this is just raw js, I assume we have access to `process.platform` and `process.env` ? maybe that's easier than json embedding?

what you have is fine tho, just a non-blocking thought